ner/src/main/java/edu/illinois/cs/cogcomp/ner/ExpressiveFeatures/BrownClusters.java

```diff
@@ -62,42 +66,38 @@ private BrownClusters() {
-                    new boolean[isLowercaseBrownClusters.size()];
-            brownclusters.wordToPathByResource = new ArrayList<>();
-            brownclusters.resources = new ArrayList<>();
-            for (int i = 0; i < pathsToClusterFiles.size(); i++) {
-                THashMap<String, String> h = new THashMap<>();
-                InFile in =
-                        new InFile(ResourceUtilities.loadResource(pathsToClusterFiles.elementAt(i)));
-                String line = in.readLine();
-                int wordsAdded = 0;
-                while (line != null) {
-                    StringTokenizer st = new StringTokenizer(line);
-                    String path = st.nextToken();
-                    String word = st.nextToken();
-                    int occ = Integer.parseInt(st.nextToken());
-                    if (occ >= thresholds.elementAt(i)) {
-                        h.put(word, path);
-                        wordsAdded++;
-                    }
-                    line = in.readLine();
-                }
 
-                if (ParametersForLbjCode.currentParameters.debug) {
-                    logger.info(wordsAdded + " words added");
+        brownclusters = new BrownClusters();
+        brownclusters.isLowercaseBrownClustersByResource =
+                new boolean[isLowercaseBrownClusters.size()];
+        brownclusters.wordToPathByResource = new ArrayList<>();
+        brownclusters.resources = new ArrayList<>();
+        for (int i = 0; i < pathsToClusterFiles.size(); i++) {
+            THashMap<String, String> h = new THashMap<>();
+            InFile in =
+                    new InFile(ResourceUtilities.loadResource(pathsToClusterFiles.elementAt(i)));
+            String line = in.readLine();
+            int wordsAdded = 0;
+            while (line != null) {
+                StringTokenizer st = new StringTokenizer(line);
+                String path = st.nextToken();
+                String word = st.nextToken();
+                int occ = Integer.parseInt(st.nextToken());
+                if (occ >= thresholds.elementAt(i)) {
+                    h.put(word, path);
+                    wordsAdded++;
                 }
-                brownclusters.wordToPathByResource.add(h);
-                brownclusters.isLowercaseBrownClustersByResource[i] =
-                        isLowercaseBrownClusters.elementAt(i);
-                brownclusters.resources.add(pathsToClusterFiles.elementAt(i));
-                in.close();
+                line = in.readLine();
+            }
+
+            if (ParametersForLbjCode.currentParameters.debug) {
+                logger.info(wordsAdded + " words added");
             }
+            brownclusters.wordToPathByResource.add(h);
+            brownclusters.isLowercaseBrownClustersByResource[i] =
+                    isLowercaseBrownClusters.elementAt(i);
+            brownclusters.resources.add(pathsToClusterFiles.elementAt(i));
+            in.close();
         }
     }
```

Comment:
Doesn't it revert the changes made in #285 for synchronizing brown cluster access?